books/free-programming-books-pt_BR.md

```diff
@@ -226,6 +226,7 @@
 
 * [Imergindo na JVM](https://leanpub.com/imergindo-jvm) - Otavio Santana *(Leanpub account or valid email requested)*
 * [Introdução a Ciência da Computação com Java](http://ccsl.ime.usp.br/files/publications/files/2008/intro-java-cc.pdf) Alfredo Goldman, Fabio Kon, Paulo J. S. Silva (PDF)
+* [Java Básico e Oriêntação a Objeto](https://canal.cecierj.edu.br/012016/d7d8367338445d5a49b4d5a49f6ad2b9.pdf) - Fundação CECIERJ (PDF)
```

Comment:
I think it might be better to credit the authors directly.

**Authors:** Clayton Escouper das Chagas, Cássia Blondet Baruque, Lúcia Blondet Baruque

Reply:
Ok

Reply:
done @Thenlie, make a merge ?